Server/app.js

```diff
@@ -0,0 +1,78 @@
+const cors = require("cors");
+const express = require("express");
+const app = express();
+
+app.use(cors());
+app.use(express.json());
+
+const logger = require("./logger");
+app.use(logger);
+
+const words = require("./words.json")
+
+//Welcome page for server
+app.get("/", (req, res, next) => {
+    res.status(200).send("This is the server for our spelling game.");
+});
+
+//List all words in the wordbank
+app.get("/words", (req, res) => {
+    res.status(200).send(words);
```

Comment:
if (words == undefined) {
        res.status(404).send();
        next();
    } else {
        res.send(words);

Reply:
Thanks and updated the PR.